src/test/scala/mesosphere/marathon/api/akkahttp/AppsDirectivesTest.scala

```diff
@@ -0,0 +1,45 @@
+package mesosphere.marathon
+package api.akkahttp
+
+import akka.http.scaladsl.model.Uri
+import akka.http.scaladsl.model.Uri.{ Path, Query }
+import akka.http.scaladsl.server.MalformedQueryParamRejection
+import akka.http.scaladsl.server.PathMatcher.{ Matched, Unmatched }
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import mesosphere.UnitTest
+import mesosphere.marathon.state.{ AppDefinition, PathId }
+import mesosphere.marathon.test.GroupCreation
+
+class AppsDirectivesTest extends UnitTest with ScalatestRouteTest {
+  import AppsDirectives._
+
+  def objectName(obj: AnyRef) = obj.getClass.getName
```

Comment:
I'm not a big fan of this way but since I cannot come up with a better one this'll do.